todo_app/data/trello_items.py

```diff
@@ -0,0 +1,121 @@
+from flask import session
+import requests, json
+
+# Define a class for Card type 
+class Item:
+        def __init__(self, id, name, statusid, status = 'To Do'):
+            self.id = id
+            self.name = name
+            self.statusid = statusid
+            self.status = status
+
+'''   
+        @classmethod
+        def from_trello_card(cls, id, name, list):
+           return cls( id, name, list )
+'''
+
+
+def get_items(url, trellobid, apikey, apitoken):
+    """
+    Fetches all cards from ToDo list on Trello.
+
+    Returns:
+        list: The list of Cards in All lists on Trello Board.
+    """
+    getcardsurl = url+"1/boards/"+trellobid+"/lists?cards=open&key="+apikey+"&token="+apitoken
+    headers = {
+        'Accept': "application/json"
+    }
+    r = requests.get(url = getcardsurl, headers=headers)
+    carddata = r.json()
+    card_nums = len(carddata)
+    all_cards = []
+    if r.status_code == 200:
+        for num in range(card_nums):
+            card_type = carddata[num]['name']
+            card_type_id = carddata[num]['id']
+            no_of_cards = len(carddata[num]['cards'])
+            for cno in range(no_of_cards):
+                cname = carddata[num]['cards'][cno]['name']
+                cid = carddata[num]['cards'][cno]['id']
+                all_cards.append( Item(cid, cname, card_type_id, card_type) )
```

Comment:
As before, if we looped over the cards directly this might be a bit easier to parse, it could then look something like:
```suggestion
            for card in cards:
                cname = card['name']
                cid = card['id']
                # note slightly different variable names for the lists section
                all_cards.append( Item(cid, cname, list_id, list_name) )
```